### PR TITLE
data :rishu6263

### PR DESCRIPTION
**Pull request by rishu6263**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7307"></a>

### CODE REVIEW

File: data/rishu6263.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+"name": "Rishabh Dwivedi",
+"displayStatsPublic": true,
+"type": "personal",
```
Comment (khushalarora) [inline data/rishu6263.json:4]: You can remove these lines, these are not required now :+1: 
```Suggestion
```

File: data/rishu6263.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+"bio": "I am web devlopment enthusiast",
+"avatar": "https://github.com/rishu6263.png",
```
Comment (khushalarora) [inline data/rishu6263.json:6]: You can also remove this line, it is not required now :+1: 
```Suggestion
```

File: data/rishu6263.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+{
+"name": "Follow me on Twitter",
+"url": "https://twitter.com/Rishabhd626312?t=-gYCEcPEC2Px3jR90BsGwQ&s=09",
+"icon": "FaTwitter"
+},
+{
+"name": "Follow me on Github",
+"url": "https://github.com/rishu6263",
+"icon": "FaGithubAlt"
+},
```
Comment (khushalarora) [inline data/rishu6263.json:17]: Please remove this extra **'Comma'** from this line :+1: 
```Suggestion
}
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @rishu6263, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**